FrontPage/Client.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+import Apollo
+import AppAuth
+
+let config = Config()
+let syncUrl = config.getConfiguration("sync-app")
+let syncConfig = syncUrl.config ?? ["error": JSONValue.bool(false)]
+let syncWs = syncConfig["websocketUrl"]?.getString() ?? "error";
```

Comment:
Generally speaking, global variables are discouraged. Especially if there are many of them and they are scattered all around different places. If you left the project and then comes back, it will be hard to remember where a variable is defined. So it's best to move them into class, or in their own classes. Here I think it's ok to move them into the `Client` class.

Reply:
+1